doc/cygnus-ngsi/installation_and_administration_guide/configuration_examples.md

```diff
@@ -1,47 +1,470 @@
+
+Nevertheless, what happens if we want, for instance, to enhance the performance by adding a sink per service? Or if we want to decide applying Name Mappings in a per service way, too? In those cases, a single super sink is not enough.
+
+Next sections explain how to achieve that by implementing advanced configurations.
+
+[Top](#top)
+
+#### <a name="section2.1.1"></a>Multiple agents
+The first (and obvious) solution is to instantiate multiple Cygnus agents (multiple JVM processes, each one listening on a dedicated TCP port), one per FIWARE service. This solution ensures each FIWARE service has specific Name Mappings, dedicated sink (or sinks), etc.
+
+```
++--------------------------------------------------------------+
+|  +--------------+     +-------------+     +---------------+  |
+|  | src-client-1 |-----| ch-client-1 |-----| sink-client-1 |  |
+|  +--------------+     +-------------+     +---------------+  |
++--------------------------------------------------------------+
+
++--------------------------------------------------------------+
+|  +--------------+     +-------------+     +---------------+  |
+|  | src-client-2 |-----| ch-client-2 |-----| sink-client-2 |  |
+|  +--------------+     +-------------+     +---------------+  |
++--------------------------------------------------------------+
+                          
+                              ...
+                          
++--------------------------------------------------------------+
+|  +--------------+     +-------------+     +---------------+  |
+|  | src-client-N |-----| ch-client-N |-----| sink-client-N |  |
+|  +--------------+     +-------------+     +---------------+  |
++--------------------------------------------------------------+
+```
+
+For each client/FIWARE service called `<id>` an `agent_<id>.conf` configuration file is required:
+
+```
+$ cat /usr/cygnus/conf/agent_<id>.conf
+# declarations
+cygnus-ngsi-<id>.sources = src-client-<id>
+cygnus-ngsi-<id>.sinks = sink-client-<id>
+cygnus-ngsi-<id>.channels = ch-client-<id>
+
+# sources
+cygnus-ngsi-<id>.sources.src-client-<id>.type = http
+cygnus-ngsi-<id>.sources.src-client-<id>.channels = ch-client-<id>
+cygnus-ngsi-<id>.sources.src-client-<id>.port = <client_port>
+cygnus-ngsi-<id>.sources.src-client-<id>...
+
+# sinks
+cygnus-ngsi-<id>.sinks.sink-client-<id>.type = ...
+cygnus-ngsi-<id>.sinks.sink-client-<id>.channel = ch-client-<id>
+cygnus-ngsi-<id>.sinks.sink-client-<id>...
+
+# channels
+cygnus-ngsi-<id>.channels.ch-client-<id>.type = ...
+cygnus-ngsi-<id>.channels.ch-client-<id>...
+```
+
+[Multi-instance](ref) support is available form the very begining of Cygnus, through the `cygnus_instance_<id>.conf` configuration files used when running Cygnus [as a service](ref):
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1420/commits/ea511e5c58c65fc69304692013f33e82ae986f1a